src/tmle_inputs/from_actors.jl

```diff
@@ -12,6 +12,24 @@ function treatments_from_actors(bqtl_file, env_file, trans_actors_prefix)
 end
 
 
+tf_error() = throw(ArgumentError("No TF specified to filter SNPs on"))
+missing_snps_error() = throw(ArgumentError("No SNPs provided to filter on"))
+
+filter_snps_by_tf(df::Nothing, tf_name::Nothing) = nothing
+filter_snps_by_tf(df::DataFrame, tf_name::Nothing) = tf_error()
```

Comment:
I am not sure what these (or the above) cover?

Reply:
I was just trying to cover edge cases in case the function didn't get passed the correct arguments, but I guess this may not happen?

Reply:
yes I don't see how this would be happening at the moment

Reply:
addressed: e4c2e131323f5032ff8a3be17776638ec0b1ba05 

However - now there is a version of this function where tf_name::Nothing is passed in and used. This happens when there is no TF in the column names of the input bqtls.csv or trans_actors.csv files.